src/Workspaces/Core/Portable/Execution/Serialization/Checksum.cs

```diff
@@ -73,8 +75,8 @@ public override int GetHashCode()
                     hash = Hash.Combine((int)_checkSum[i], hash);
                 }
 
-                _lazyHash = hash;
-            }
+                _lazyHash = new StrongBox<int>(hash);
+            };
```

Comment:
hmmm you can search _lazy in our code base. there are just so many places.

like this
_lazyAllLocals = new ArrayBuilder<Cci.ILocalDefinition>(1);
or this
_lazyAssemblyReferences = this.MetadataReader.GetReferencedAssembliesOrThrow();

just search _lazy and you will bunch of them...